app/konnect/getting-started/migration.md

```diff
@@ -0,0 +1,318 @@
+
+* [{{site.kic_product_name}} (KIC)](/kubernetes-ingress-controller/latest/)
+    
+    If you run KIC on premises, migrating to Konnect is straightforward. 
+    The [Kong Ingress Controller for Kubernetes Assocation](/konnect/gateway-manager/kic/)
+    documentation provides details on linking your KIC deployment to a KIC based Control Plane
+    in {{site.konnect_product_name}}.
+
+* [DB-less mode](/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/)
+
+    If you're running a DB-less deployment and interested in migrating to Konnect, Kong recommends migrating to 
+    Hybrid mode deployment to take advantage of the full capabilities of Kong Konnect. Please reach out to
+    a Kong representative for assistance with this migration.
+
+### Role Based Access Controls
+
+Both {{site.base_gateway}} and {{site.konnect_product_name}} provide frameworks for controlling security and
+data access. 
+
+{{site.base_gateway}} provides a traditional Role Based Access Control (RBAC) system
+to manage administrators and users of the API gateway system. {{site.konnect_product_name}} provides a robust 
+multi-layer Identity and Access Management (IAM) system including organizations, teams and roles. 
+Konnect also provides integrations with IdP providers via OIDC allowing you to map centrally managed teams to 
+Konnect based roles.
+
+{{site.base_gateway}}'s RBAC system does not map directly to the IAM system provided by {{site.konnect_product_name}}. 
+Users migrating from {{site.base_gateway}} on-premises to {{site.konnect_product_name}} have typically 
+chosen to use Konnect's IdP integrations and take advantage of their existing IdP solution and Konnect team 
+based mappings instead migrating their {{site.base_gateway}} on-premises RBAC configuration directly.
+
+The following are the general steps for setting up IAM in {{site.konnect_product_name}} for your migration:
+
+* [Sign up](https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs) for 
+  Konnect (if necessary), and use the [Org Switcher](/konnect/org-management/org-switcher/) 
+  to create or select the Organization you are going to migrate your on-premises deployment to. 
+* [Set up single sign-on (SSO) access to Konnect using an existing IdP provider](/konnect/org-management/oidc-idp/)
+* [Create teams](/konnect/org-management/teams-and-roles/manage/) in Konnect or use 
+  [predefined teams](/konnect/org-management/teams-and-roles/teams-reference/) to create your desired organizational structure.
+* For any custom teams, [assign the appropriate roles](/konnect/org-management/teams-and-roles/manage/#edit-roles-for-a-team) 
+  from the predefined list of available roles in Konnect.
+* Use the Konnect IdP Team Mappings feature to 
+  [map the Konnect teams to your IdP provider groups](/konnect/org-management/okta-idp/#map-konnect-teams-to-okta-groups).
+
+### Workspaces -> Control Planes
```

Comment:
```suggestion
## Migrating from Workspaces to control planes
```

I would make this an H2, and make the subtopics H3's. The information ehre is solid enough to stand alone.

Reply:
Let's hold off on changing header levels for now. Let's leave it nested, along with RBAC, inside the migration guide for this first iteration. I do agree with the retitling though.
```suggestion
### Migrating from workspaces to control planes
```